src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsFetchingSpec.groovy

```diff
@@ -1575,6 +1575,38 @@ class PriceFloorsFetchingSpec extends PriceFloorsBaseSpec {
         invalidSkipRate << [MIN_SKIP_RATE - 1, MAX_SKIP_RATE + 1]
     }
 
+    def "PBS should reject fetch when response header is invalid"() {
+        given: "Test start time"
+        def startTime = Instant.now()
+
+        and: "Default BidRequest"
+        def bidRequest = BidRequest.defaultBidRequest
+
+        and: "Account with enabled fetch, fetch.url in the DB"
+        def accountId = bidRequest.site.publisher.id
+        def account = getAccountWithEnabledFetch(accountId)
+        accountDao.save(account)
+
+        and: "Set Floors Provider response with header"
+        def floorsResponse = PriceFloorData.priceFloorData
+        def headerKey = "Cache-Control"
+        def header = [(headerKey): "no-cache, no-store, max-age=0, must-revalidate"]
```

Comment:
Please create a test to make sure that PBS honors this parameter by redownloading rules after max age.